### PR TITLE
Add optional `--identity` flag to manually set code signing identity

### LINKED ISSUE #23: Codesign identity as optional parameter
For two Developer ID distribution certificates installed on my machine (both from different teams) I receive following warning:

```
Code signing failed. The DMG is fine, just not code signed.
Developer ID Application: ambiguous (matches "Developer ID Application: OrganizationA (XXXXXXXXXX)" and "Developer ID Application: OrganizationB (YYYYYYYYY)" in /Users/user/Library/Keychains/login.keychain-db)
XXXXXXXXXX
```
It would be nice to have an optional parameter to resolve this issue, for example:

`
--identity "Developer ID Application: OrganizationB (YYYYYYYYY)"
`

### PR DESCRIPTION
**Pull request by dminta**
Fixes #23

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -116,11 +120,15 @@ ee.on('finish', async () => {
 	try {
 		let identity;
 		const {stdout} = await execa('security', ['find-identity', '-v', '-p', 'codesigning']);
-		if (stdout.includes('Developer ID Application:')) {
+		if (cli.flags.identity && stdout.includes('"' + cli.flags.identity + '"')) {
```
Comment (sindresorhus) [inline cli.js]:
```suggestion
		if (cli.flags.identity && stdout.includes(`"${cli.flags.identity}"`)) {
```

File: cli.js [hunk 2]
```diff
@@ -116,11 +120,15 @@ ee.on('finish', async () => {
 	try {
 		let identity;
 		const {stdout} = await execa('security', ['find-identity', '-v', '-p', 'codesigning']);
-		if (stdout.includes('Developer ID Application:')) {
+		if (cli.flags.identity && stdout.includes(`"${cli.flags.identity}"`)) {
```
Comment (sindresorhus) [inline cli.js]: Actually, I don't think it should just silently fall back to automatic if the passed in identity does not exist. It should instead error with a user-friendly error message.

Comment (dminta) [reply cli.js]: In a given case we always receive `No usable identity found` error - is it fine or message should be different?

**Comment by Bargsteen** [comment]
Any progress on this? It would be a great feature :)

File: cli.js (hunk 2 above)
Comment (sindresorhus) [reply cli.js]: Let's go with:

> No suitable code signing identity found

**Comment by sindresorhus** [comment]
@dminta Bump :)

**Comment by dminta** [comment]
Currently I'm not an everyday mac user; I will finish the feature in this or next week.

**Comment by dminta** [comment]
@sindresorhus OK, I guess it's ready to merge.

**Pull request merged**